app/views/includes/scripts.html

```diff
@@ -10,4 +10,8 @@
 
 {% if enableTracking === 'true' %}
 <script src="/public/javascripts/gatracking.js"></script>
-{% endif %}
\ No newline at end of file
+{% endif %}
+
+<!--[if IE 8]>
```

Comment:
We've got this as a bug for IE8 but `formaction` is also not supported by IE9.

Have you tested this with IE9? If not, can you do that to see if we need to enable the polyfill for that browser too?